specificatie/openapi.yaml

```diff
@@ -61,14 +61,13 @@ paths:
             -  verblijfplaats__nummeraanduidingIdentificatie
 
       parameters:
-        - name: expand
-          deprecated: true
-          in: query
-          required: false
-          description: "Hiermee kun je opgeven welke gerelateerde resources meegeleverd moeten worden, en hun inhoud naar behoefte aanpassen. Hele resources of enkele properties geef je in de expand parameter kommagescheiden op. Properties die je wil ontvangen geef je op met de resource-naam gevolgd door de property naam, met daartussen een punt. In de definitie van het antwoord kun je bij _embedded zien welke gerelateerde resources meegeleverd kunnen worden. Zie [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/expand.feature)."
+        - in: query
+          name: fields
+          required: true
+          description: "Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een door komma's gescheiden lijst van property namen op te geven. Bij opgave van niet-bestaande properties wordt een 400 Bad Request teruggegeven. De fiels parameter is verplicht vanwege dataminimalisatie overwegingen. Zie [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature)"
           schema:
             type: string
-        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields"
+            pattern: ^[a-zA-Z.,]+$
```

Comment:
aan het pattern moet je denk ik ook _ (underscore) toevoegen, zodat je ook specifiek om _links kan vragen. Is misschien niet direct noodzakelijk (je kan ook _links.adres vragen met fields=adres), maar wel completer